src/pages/EventDetailsPage/components/DeleteEditButtons/index.js

```diff
@@ -30,26 +32,28 @@ const DeleteEditButtons = props => {
 
   const confirmButtonProps = {
     name: 'Yes',
-    onClick: handleConfirmDelete,
-    to: ROUTES.CALENDAR,
-    component: Link,
+    actionCallback: () => {
+      handleConfirmDelete();
+      history.push(ROUTES.CALENDAR);
+    },
+    primary: true,
     positive: true,
   };
 
   const cancelButtonProps = {
     name: 'No',
+    primary: true,
     positive: true,
   };
 
   return (
     <div className={classes.root}>
       <ButtonWithConfirmationModal
-        title='Delete this event?'
-        contentText='Do you want to delete this event permanently?'
+        modalTitle='Delete this event?'
+        modalContentText='Do you want to delete this event permanently?'
+        name='Delete'
         confirmButtonProps={confirmButtonProps}
```

Comment:
looking at this usage now, maybe modalConfirmButtonProps would make more sense? up to you for that though